chrysler_pacifica_ehybrid_s_2019/VehicleConfigParams.yaml

```diff
@@ -42,3 +42,6 @@ vehicle_deceleration_limit: 6.0
 # Units: (m^-1) / s
 vehicle_max_curvature_rate: 0.75
 
+# State machine type
+# type: lexus or cadilac
+vehicle_state_machine_type : lexus
```

Comment:
I think we should probably rename these state machines from lexus and cadillac. 
How about instead of cadillac we call it something like HardwareEngagedAutomation
and instead of Lexus we call it SoftwareOnlyEngagedAutomation.

You would want to update the cadillac and lexus class names in CARMAPlatform as well.